Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -1725,6 +1814,8 @@ private static T FindImmediateChild<T>(HierarchyNode parent)
                         return "e,env,environment: p,package: a,admin";
                     case PythonConstants.GenerateRequirementsTxt:
                         return "e,env,environment:";
+                    case PythonConstants.ProcessRequirementsTxt:
+                        return "path";
```

Comment:
IIRC, you need the trailing colon to indicate that the parameter takes a value. You can test this from the Command Window.